internal/constants/constants.go

```diff
@@ -4,11 +4,13 @@ package constants
 const (
 	QueueMessageTypeTask      = "task"
 	QueueMessageTypeHandshake = "handshake"
+	QueueMessageTypeStatus    = "status"
 )
 
-// Queue names
+// Queue constants
 const (
 	WorkerQueueName = "worker_queue"
+	MaxWorkers      = 10
```

Comment:
This should be moved to config, and be parsed form ENV. Variables which might change from system to system are "configurable", so should be moved to config. Constants, as per definition rather will never change